.github/workflows/update-version.yml

```
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2025 Canonical Ltd.
+name: Update Version
+
+on:
+  workflow_run:
+    workflows: ["Tag GitHub"]
```

Does this have to use brackets or can it be a list such as below?
```suggestion
    workflows:
        - "Tag GitHub"
```